.github/workflows/e2e_test_ci.yml

```diff
@@ -516,6 +519,10 @@ jobs:
         run: |
           killall spice || true
           cat spice.log
+          if grep -iE "(failed|error)" spice.log; then
```

Comment:
Initial load can be successful but refreshes can error so we check logs to identify such issues